Prepare simple Grafana deployment (#19)
appuio/appuio-cloud-reporting: README.md

kubectl -n appuio-reporting apply -k manifests/base
```

### Grafana

There is a Grafana deployment prepared under `manifests/grafana`.

> To be able to use that one, customize the parameters in `grafana-helm-values.yaml` and run `make` to generate the manifest.

Reviewer: ```suggestion
To be able to use the deployment, customize the parameters in `grafana-helm-values.yaml` and run `make` to generate the manifest.
```

> The Grafana Helm Chart is coming from https://grafana.github.io/helm-charts.

Reviewer: ```suggestion
Add the required Grafana Helm chart using `helm repo add grafana https://grafana.github.io/helm-charts`.
```

> A secret is needed:

Reviewer: ```suggestion
The deployment requires a secret `grafana-creds` containing the admin username and password:
```
